backend/src/server/createServer.ts

```diff
@@ -52,6 +55,11 @@ const createServer = async (
     throw new Error('Fatal: Database Version incorrect')
   }
 
+  // start DHT hyperswarm when DHT_TOPIC is set in .env
+  if (CONFIG.DHT_TOPIC) {
+    await startDHT(con, CONFIG.DHT_TOPIC)
+  }
+
```

Comment:
As i understand this here, the hyperswarm will be in the first steps part of our apollo-server.
In the future we should extract it as a separated modul, which will communicate with the existing moduls, right?